pkg/services/apiserver/restconfig/directly_serve_middleware.go

```diff
@@ -0,0 +1,45 @@
+package restconfig
```

Comment:
This is the main file which was added here, and its use by `featuremgmt` brought in the import cycles, which forced the bigger refactor.